src/Microsoft.SqlTools.ServiceLayer/FileBrowser/FileValidator/DisasterRecoveryFileValidator.cs

```diff
@@ -0,0 +1,143 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Data;
+using System.Globalization;
+using System.IO;
+using Microsoft.SqlServer.Management.Common;
+using Microsoft.SqlServer.Management.Sdk.Sfc;
+using Microsoft.SqlServer.Management.Smo;
+using System.Data.SqlClient;
+
+namespace Microsoft.SqlTools.ServiceLayer.FileBrowser.FileValidator
+{
+    /// <summary>
+    /// Validate selected file paths for backup/restore operations
+    /// </summary>
+    public class DisasterRecoveryFileValidator : IFileValidator
```

Comment:
should this logic reside in the disasterrecovery directory somehere?

Reply:
moved it to disasterrecovery